server/index-node/Cargo.toml

```diff
@@ -13,4 +13,4 @@ graphql-parser = "0.4.0"
 http = "0.2"
 hyper = "0.14"
 serde = "1.0"
-either = "1.6.1"
+either = "1.6.1"
```

Comment:
What happened to the empty newline, some funkiness from your editor?

Reply:
Yeah for sure, I'll fix that :)